xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -807,6 +807,12 @@ void CGUIWindowMusicBase::GetContextButtons(int itemNumber, CContextButtons &but
         {
             buttons.Add(CONTEXT_BUTTON_PLAY_PARTYMODE, 15216); // Play in Partymode
         }
+        if (item->IsAudioBook())
+        {
+          int bookmark;
+          if (m_musicdatabase.GetResumeBookmarkForAudioBook(item->GetPath(), bookmark) && bookmark > 0)
+            buttons.Add(CONTEXT_BUTTON_RESUME_ITEM, 39011);
```

Comment:
Will only work in this window. New context menus should implement IContextMenuItem and added globally.
(and accessing db in the visibility check is a no-go. this info should be attached to the listitem when created)